extensions/common/json_util.cc

```diff
@@ -20,60 +20,91 @@
 namespace Wasm {
 namespace Common {
 
-::nlohmann::json JsonParse(absl::string_view str) {
-  return ::nlohmann::json::parse(str, nullptr, false);
+std::pair<JsonObject, JsonParserResultDetail> JsonParse(absl::string_view str) {
```

Comment:
Can we return just the JsonObject as the original logic instead of a pair? I don't see the needs of returning detail here. `is_object` on returned object should already represent that.

Reply:
I don't agree with this. It seems that `is_object()` can't handle if that is empty or discarded. So we should use this function with basic `PARSE_ERROR` and `is_object()` to validate json strictly.

Reply:
But I agree that the usage of this function is pretty redundant. We should handle that is object or not with `JsonParserResultDetail`.

Reply:
Ok, yeah, that's do that then.